include/arch/arm/aarch64/timer.h

```diff
@@ -45,6 +46,22 @@ static ALWAYS_INLINE void arm_arch_timer_enable(unsigned char enable)
 			 : : "r" (cntv_ctl) : "memory");
 }
 
+static ALWAYS_INLINE void arm_arch_timer_irq_mask(unsigned char mask)
+{
+	uint32_t cntv_ctl;
+
+	__asm__ volatile("mrs %0, cntv_ctl_el0\n\t"
+			 : "=r" (cntv_ctl) :  : "memory");
+
+	if (mask)
+		cntv_ctl |= CNTV_CTL_IMASK;
+	else
+		cntv_ctl &= ~CNTV_CTL_IMASK;
```

Comment:
Unless there has been a change in the coding guidelines (please do let me know if there has; I've been away for a while), these should have curly braces.

```suggestion
	if (mask) {
		cntv_ctl |= CNTV_CTL_IMASK;
	} else {
		cntv_ctl &= ~CNTV_CTL_IMASK;
	}
```

Reply:
@stephanosio glad you are back :)